lib/site_template/index.md

```diff
@@ -1,6 +1,15 @@
 ---
-# You don't need to edit this file, it's empty on purpose.
-# Edit theme's home layout instead if you wanna make some changes
-# See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+# This file is empty because by default, the "content" is adequately realised
```

Comment:
I feel like this line is just going to confuse newcomers even more. What about replacing this with the original `You don't need to edit this file, it's empty on purpose` or some variation of it? If a user wants more information, they can look at the URL provided later on.

Reply:
I specifically want to do away with `You don't need to edit this file, it's empty on purpose`. It sounds like we're imposing something on the user..

Reply:
I like zis.. :+1: